connection_test.clj

```diff
@@ -0,0 +1,350 @@
+(ns metabase.driver.sql-jdbc.connection-test
```

Comment:
So what changed in this file?

Reply:
I made the `test-ssh-tunnel-connection` test into a no-op